.changeset/gorgeous-comics-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pagopa/typescript-config-node": minor
+---
+
+[CES-581] Add typescript configurations for Node v22
```

Comment:
why are we using `node 22` ?

Reply:
why not?

Reply:
because it's the latest LTS version available. why not using it?

are there any issues with the dx workflows if we use node 22?

Reply:
Because it's not yet in `GA` on Azure, but only in `Preview`. According to the Azure roadmap, it will be general available form the Q2 2025.

Reply:
I left here the reference of the roadmap, so other people can easily see it.

https://github.com/Azure/azure-functions-nodejs-library/wiki/Roadmap